app/helpers/application_helper/toolbar/template_infras_center.rb

```diff
@@ -1,137 +1,137 @@
+          N_('Manage Policies'),
+          :url_parms => "main_div",
+          :enabled   => false,
+          :onwhen    => "1+"),
+        button(
+          :miq_template_policy_sim,
+          'fa fa-play-circle-o fa-lg',
+          N_('View Policy Simulation for the selected Templates'),
+          N_('Policy Simulation'),
+          :url_parms => "main_div",
+          :enabled   => false,
+          :onwhen    => "1+"),
+        button(
+          :miq_template_tag,
+          'pficon pficon-edit fa-lg',
+          N_('Edit tags for the selected Templates'),
+          N_('Edit Tags'),
+          :url_parms => "main_div",
+          :enabled   => false,
+          :onwhen    => "1+"),
+        button(
+          :miq_template_check_compliance,
+          'fa fa-search fa-lg',
+          N_('Check Compliance of the last known configuration for the selected Templates'),
+          N_('Check Compliance of Last Known Configuration'),
+          :url_parms => "main_div",
+          :confirm   => N_("Initiate Check Compliance of the last known configuration for the selected Templates?"),
+          :enabled   => false,
+          :onwhen    => "1+"),
+      ]
+    ),
+  ])
   button_group('miq_template_lifecycle', [
-                 select(
-                   :miq_template_lifecycle_choice,
-                   'fa fa-recycle fa-lg',
-                   t = N_('Lifecycle'),
-                   t,
-                   :items => [
-                     button(
-                       :miq_template_miq_request_new,
-                       'pficon pficon-add-circle-o fa-lg',
-                       N_('Select a single Template to Provision VMs'),
-                       N_('Provision VMs using selected Template'),
-                       :url_parms => "main_div",
-                       :enabled   => false,
-                       :onwhen    => "1"),
-                     button(
-                       :miq_template_clone,
-                       'product product-clone fa-lg',
-                       N_('Clone this Template'),
-                       N_('Clone selected Template'),
-                       :url_parms => "main_div",
-                       :enabled   => false,
-                       :onwhen    => "1"),
-                   ]
-                 ),
-               ])
+    select(
+      :miq_template_lifecycle_choice,
+      'fa fa-recycle fa-lg',
+      t = N_('Lifecycle'),
+      t,
+      :items => [
+        button(
+          :miq_template_miq_request_new,
+          'pficon pficon-add-circle-o fa-lg',
+          N_('Select a single Template to Provision VMs'),
+          N_('Provision VMs using selected Template'),
+          :url_parms => "main_div",
+          :enabled   => false,
+          :onwhen    => "1"),
+        button(
+          :miq_template_clone,
+          'product product-clone fa-lg',
+          N_('Clone this Template'),
+          N_('Clone selected Template'),
+          :url_parms => "main_div",
```

Comment:
I'd remove the toolbar in separate PR, if I won't be able to find, where it is rendered.
@h-kataria do you have any idea if this toolbar is still used?